tests/test_signal.py

```diff
@@ -16,13 +16,16 @@ def test_signal_init_list(impulse_list):
     signal = Signal(impulse_list, 44100, domain='time')
     assert isinstance(signal, Signal)
 
+    signal = Signal(impulse_list, 44100, domain='freq')
+    assert isinstance(signal, Signal)
+
 
 def test_signal_init_default_parameter(impulse_list):
     # using all defaults
     signal = Signal(impulse_list, 44100)
     assert signal.domain == 'time'
     assert signal.fft_norm == 'none'
-    assert signal.comment is None
+    assert signal.comment == 'None'
```

Comment:
```suggestion
    assert signal.comment == 'none'
```
Let's keep this consistent and better distinguishable from None

Reply:
Done as suggested :)